### DISCUSSION TITLE
`nerd-fonts-jetbrains-mono` or `ttf-jetbrains-mono-nerd`

### DISCUSSION THREAD

**Discussion by popjdh** [ayamir/nvimdots#909] [General]
There are two packages in the archLinux repository that provide patched font JetBrains Mono.
```
❯ yay ttf-jetbrains-mono-nerd
1 extra/ttf-jetbrains-mono-nerd 3.0.2-1 (6.1 MiB 184.1 MiB) [nerd-fonts] (Installed)
    Patched font JetBrains Mono from nerd fonts library
==> Packages to install (eg: 1 2 3, 1-3 or ^4)
==>
 there is nothing to do
❯ yay nerd-fonts-jetbrains-mono
2 aur/nerd-fonts-jetbrains-mono-160 2.1.0-1 (+1 0.43) (Out-of-date: 2022-08-26)
    A Nerd Font patched version of JetBrains Mono, version with patched nbsp
1 archlinuxcn/nerd-fonts-jetbrains-mono 2.2.2-1 (1.7 MiB 64.9 MiB)
    JetBrainsMono Nerd Font: patched from JetBrains Mono with Nerd Fonts Glyphs(Icons)
==> Packages to install (eg: 1 2 3, 1-3 or ^4)
==>
 there is nothing to do
```
Compared to the version of archlinuxcn source, the version of extra source is more recent.

Are there any considerations when choosing the version of archlinuxcn source ?

**Comment by kang8** [comment]
Use `extra/ttf-jetbrains-mono-nerd 3.0.2-1`.

`aur/nerd-fonts-jetbrains-mono-160` is out of date in your output info. And nerd-font has been released to [v3](https://github.com/ryanoasis/nerd-fonts/releases/tag/v3.0.0)

**Reply by popjdh** [reply]
According to the release information of v3, it seems that there are some code point changes on characters after v2.3.0 The version of archlinuxcn source is the previous version.

It is unclear whether the author chose an earlier version due to this change.

**Reply by kang8** [reply]
nvimdots has been upgraded nerd-font to v3, see #447 for more information.

**Reply by ayamir** [reply]
Edited. I'm using Gentoo now so the arch package is updated infrequently. Please feel free to open more discussions if you find any out-of-date info in wiki :+1: . I will update them.

**Reply by popjdh** [reply]
@kang8 Thank you😊. I forgot to check the relevant information of the release and only saw the content of the wiki.
